torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -182,7 +182,7 @@ def insertable_tensor_check(self, t: torch.Tensor) -> bool:
         # TODO - we could also Tensors which get replaced with arange here
         return (
             t.numel() != 0
-            and (t == t.flatten()[0]).all()
+            and bool((t == t.flatten()[0]).all())
```

Comment:
NIT
```suggestion
            and (t == t.flatten()[0]).all().item()
```

Reply:
Hm, looks like the `bool()` would be necessary anyway as `item()` is typed as returning `Union[bool, int, float, complex]`. I guess ideally we'd have generics in our Tensor type...

Reply:
bool is pretty much the same as `item()` so I think it's alright as it is